### PR TITLE
Docs: Add Development.md

### PR DESCRIPTION
**Pull request by s-l-teichmann**
Solves #488

### CODE REVIEW

**Review by JanHoefelmeyer — DISMISSED** [review]
LGTM

File: docs/Development.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Development
+
+## Generated files
+
+Some source code files are machine generated. At the moment these are only
+[cvss20enums.go](../csaf/cvss20enums.go) and [](../csaf/cvss20enums.go) on the
```
Comment (ThomasJunk) [inline docs/Development.md:6]: the second link is ```[csaf/cvss3enums.go](../csaf/csaf/cvss3enums.go)```?

**Review by ThomasJunk — DISMISSED** [review]
LGTM (minor change suggested)

**Review by ThomasJunk — APPROVED** [review]

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM

**Pull request merged**